.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] **Required** There is a .gitignore file listing `test.md` file and `node_modules` directory (as it can be added by accident while setting up linters). Please add `node_modules` also.

Reply:
Noted, thank you